spec/fail_to_ban_spec.rb

```diff
@@ -42,111 +42,112 @@ def del(k)
 
   let(:storage) { NullStorage.new(data) }
 
-  describe '#protect' do
-
-    subject { described_class.new(storage: storage, unique_key: key).protect }
-
-    let(:key) { 'dev@jobteaser.com' }
-
-    context 'when retry < 3' do
+  describe 'BackoffStrategy' do
```

Comment:
I really think that we should still test the "core" class. If you do not see what to test or how, I'd be glad to setup a quick pairing ;)